configure.ac

```diff
@@ -24,10 +25,74 @@ AC_CONFIG_HEADERS([config.h])
 dnl Where are the sources
 AC_CONFIG_SRCDIR([src/main.c])
 
-dnl dependencies
+dnl Checks for programs
+dnl Checks for libraries
 AC_CHECK_LIB(crypto, SHA1_Init)
 
-dnl Checks for programs
+dnl Checks for headers
+AC_CHECK_HEADERS([
+	sys/ptrace.h
+	linux/ptrace.h
+	sys/signal.h
+	sys/syscall.h
+	sys/wait.h
+])
+
```

Comment:
why are these separate?

and it would be slightly more readable if the list was sorted.

Reply:
@zvr they were separated according to which file uses them, the first one is for "trace.c" and the second one for "hash.c"

I found it easier this way. But I wouldn't mind merging them.

By sorted you mean lexicographical order?

Reply:
Yes, better have a single, sorted list.
For example, if more than one files include the same header, we shouldn't be checking them twice.

Reply:
done